test/client/setup-test-framework.js

```diff
@@ -1,7 +1,9 @@
 import '@testing-library/jest-dom';
-
+const { TextEncoder, TextDecoder } = require( 'util' );
 const nock = require( 'nock' );
 
+global.TextEncoder = TextEncoder;
```

Comment:
Any chance we can set this up only for the tests that are going to need it?

Reply:
Good point!
I found a workaround by running this part in the test directly, before JSDOM.